test/py/requirements-onnx.txt

```diff
@@ -21,9 +21,10 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #####################################################################################
-
-onnx==1.14.1
+onnx==1.16;python_version>="3.12"
+onnx==1.14.1;python_version<"3.12"
 protobuf==3.20.2
-numpy==1.21.6
+numpy==1.26;python_version>="3.12"
```

Comment:
go to 1.26.4, its the release before numpy 2.0.x

Reply:
Done.